modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -130,10 +130,10 @@ public void processOpts() {
     @SuppressWarnings({ "static-method", "unchecked" })
     public Map<String, Object> postProcessAllModels(Map<String, Object> objs) {
         if (supportsInheritance) {
-            // Index all CodegenModels by name.
+            // Index all CodegenModels by model name.
             Map<String, CodegenModel> allModels = new HashMap<String, CodegenModel>();
             for (Entry<String, Object> entry : objs.entrySet()) {
-                String modelName = entry.getKey();
+                String modelName = toModelName(entry.getKey());
```

Comment:
Can you elaborate on why you used `toModelName` here? Is this for addressing a particular issue?

This change will impact all other code generator so I'm thinking if there's a better way to handle it in the Javascript code generator.